### PR TITLE
ci: replace with sepolia

### PR DESCRIPTION
**Pull request by sambacha**
all heil sepolia

### CODE REVIEW

File: .github/workflows/live-test.yml [hunk 1]
```diff
@@ -1,34 +1,40 @@
...
+    CARGO_UNSTABLE_SPARSE_REGISTRY: true
+    CARGO_INCREMENTAL: 0
 
 jobs:
     live:
         name: goerli live tests
         runs-on: ubuntu-latest
         env:
-            GOERLI_RPC_URL: https://goerli.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161
+            GOERLI_RPC_URL: https://sepolia.infura.io/v3/fe24c1ff7cfb4ddabec77f512e60c5b6
```
Comment (mattsse) [inline .github/workflows/live-test.yml:18]: should also rename this var

**Review by mattsse — CHANGES_REQUESTED** [review]
wdyt @joshieDo

File: .github/workflows/live-test.yml (hunk 1 above)
Comment (sambacha) [reply .github/workflows/live-test.yml:18]: yea I wanted to see if it works 1st

File: .github/workflows/live-test.yml [hunk 2]
```diff
@@ -1,34 +1,47 @@
...
 
-name: live-test
+permissions:
+  contents: read
 
 env:
-    CARGO_TERM_COLOR: always
+  CARGO_TERM_COLOR: always
+  CARGO_UNSTABLE_SPARSE_REGISTRY: true
+  CARGO_INCREMENTAL: 0
```
Comment (gakonst) [inline .github/workflows/live-test.yml:14]: pls remove unrelated stuff

File: .github/workflows/live-test.yml [hunk 3]
```diff
@@ -1,34 +1,47 @@
+name: live-test
+
 on:
-    push:
-        branches:
-            - master
+  push:
+    branches: ['master']
+  pull_request:
```
Comment (gakonst) [inline .github/workflows/live-test.yml:6]: pls remove unrelated changes

**Comment by sambacha** [comment]
@mattsse can we add dispatch? this wont build unless is a push onto master so not exactly ideal

**Comment by gakonst** [comment]
would like to keep thing as they are, thanks!

**Comment by sambacha** [comment]
> would like to keep thing as they are, thanks!

Close the corresponding issue then while your at it 😉

**Pull request closed without merging**